cl/assets/templates/includes/dismissible_nav_banner.html

```diff
@@ -35,29 +31,33 @@
 {% endcomment %}
 
 {% if cookie_name  not in request.COOKIES %}
-<div class="navbar navbar-default subnav alert-{% if variant %}{{ variant }}{% else %}danger{% endif %} alert-dismissible" role="navigation">
-  <div class="container-fluid">
-    <div class="row">
-      <div class="col-xs-12">
-        <button type="button" class="close"
-                data-cookie-name="{{cookie_name}}"
-                data-duration="20"
-                aria-label="Close">
-            <span aria-hidden="true"
-                  class="x-large">&times;</span></button>
-      </div>
-    </div>
-    <div class="row flex flex-column {% if single_row %}flex-sm-row{% endif %} align-items-center justify-content-between">
-      <div class="col-xs-12 {% if single_row %}col-sm-10{% endif %} navbar-text lead">
-        <p>{% if emoji %}{{emoji}}{% endif %} {{text}}</p>
-      </div>
-      <div class="{% if single_row %}col-xs-2{% else %}col-xs-12{% endif %} flex justify-content-center justify-content-sm-end {% if not single_row %}second-row-btn{% endif %}">
-        <a href="{{link}}"
-            class="btn btn-{% if variant %}{{ variant }}{% else %}danger{% endif %} btn-lg hidden-xs">{% if button_emoji %}{{button_emoji}}{% endif %}&nbsp;{% if button_text %}{{button_text}}{% else %}Learn More{% endif %}</a>
-        <a href="{{link}}"
-            class="btn btn-{% if variant %}{{ variant }}{% else %}danger{% endif %} btn-sm hidden-sm hidden-md hidden-lg">{% if button_emoji %}{{button_emoji}}&nbsp;{% endif %}{% if button_text %}{{button_text}}{% else %}Learn More{% endif %}</a>
-      </div>
-    </div>
+  <div
+    class="flex flex-column align-items-center flex-md-row alert-{% if variant %}{{ variant }}{% else %}danger{% endif %}"
+    style="border: 2px solid lightgray; padding-inline: 4rem; padding-block: 2rem; position: relative; gap: 1.5rem"
```

Comment:
Noticed this uses inline styles. we usually keep styling in `override.css` instead of adding it directly to the HTML. 

Would it make sense to move this inline style into a reusable CSS class in override.css? It’s tweaking several properties, so having it as a class might make it easier to maintain.

Reply:
I've been religious about no inline styles for a decade, but this UI is on its dying days, so I don't think it matters if it works.